waku/v2/node/waku_node.nim

```diff
@@ -258,10 +263,32 @@ proc new*(T: type WakuNode,
     peerManager: PeerManager.new(switch, peerStorage),
     switch: switch,
     rng: rng,
-    enr: enr,
+    enr: populatedEnr,
     announcedAddresses: announcedAddresses
   )
 
+  if discv5Enabled:
+    var discv5Enrs: seq[enr.Record]
+    # parse enrURIs from the configuration and add the resulting ENRs to the discv5BootstrapEnrs seq
+    for enrUri in discv5BootstrapNodes:
+      addBootstrapNode(enrUri, discv5Enrs)
+    discv5Enrs.add(discv5BootstrapEnrs)
```

Comment:
I think we can perhaps keep the node initialisation a bit cleaner by having this population done by the application? I.e. application provides seq of `enr.Record`.

Reply:
Addressed in [74c3622](https://github.com/waku-org/nwaku/pull/1512/commits/74c36223864222f0da14cee19f4a8f9b1f7f3654)